core/db/migrate/20210408092939_create_spree_menu_items.rb

```diff
@@ -0,0 +1,24 @@
+class CreateSpreeMenuItems < ActiveRecord::Migration[6.0]
+  def change
+    create_table :spree_menu_items do |t|
+      t.column :name, :string
+      t.column :subtitle, :string
+      t.column :url, :string
+      t.column :new_window, :boolean, default: false
+      t.column :item_type, :string
+      t.column :linked_resource_type, :string, default: 'URL'
+      t.column :linked_resource_id, :integer
```

Comment:
index on both fields please

Reply:
I'll `add_index :spree_menu_items, [:linked_resource_type, :linked_resource_id]` to optimise searching for a Taxon and  it's ID or a Product and it's ID, or Page and it's ID

<img width="1175" alt="Screenshot 2021-04-15 at 11 18 29" src="https://user-images.githubusercontent.com/1240766/114854114-64890780-9ddc-11eb-914f-ef4bfba29224.png">
<img width="1193" alt="Screenshot 2021-04-15 at 11 18 42" src="https://user-images.githubusercontent.com/1240766/114854127-6783f800-9ddc-11eb-9e70-f6b2c90bbd5e.png">